docs/content/en/docs/getting-started/install/_index.md

```diff
@@ -101,6 +101,62 @@ You can verify your installed version with
 eksctl anywhere version
 ```
 
+## Prepare for airgapped deployments (optional)
+
+When creating an EKS Anywhere cluster, there may be times where you need to do so in an airgapped
+environment.
+In this type of environment, cluster nodes are connected to the Admin Machine, but not to the
+internet.
+In order to download images and artifacts, however, the Admin machine needs to be temporarily
+connected to the internet.
+
+An airgapped environment is especially important if you require the most secure networks.
+EKS Anywhere supports airgapped installation for creating clusters using a registry mirror.
+For airgapped installation to work, the Admin machine must have:
+
+* Temporary access to the internet to download images and artifacts
+* Ample space (50 GB or more) to store artifacts locally
```

Comment:
We had a 50Gb admin machine in vSphere, which couldn't store all the artifacts. We increased the disk size to 80. So I think we should recommend 80Gb here.

Reply:
Fixed.